deploy.py

```diff
@@ -68,27 +68,47 @@ def deploy():
     # proxy.secretTokens have leaked. So let's be careful with that!
     PROXY_SECRET_KEY = bytes.fromhex(os.environ['PROXY_SECRET_KEY'])
 
-    clusters = parse_clusters()
-    for cluster in clusters:
+    if cluster and hub:
         with cluster.auth():
-            for hub in cluster.hubs:
-                hub.deploy(k, PROXY_SECRET_KEY)
+            hub.deploy(k, PROXY_SECRET_KEY)
```

Comment:
Cluster & Hub are strings here, right? We'd have to make them cluster & hub objects somehow.

Reply:
I just added some search functions. Sorry for missing this :(